src/services/users/reset-password/reset-password.test.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('chai')
```

Comment:
for our convention, we would want the file name to be the same as the actual file. the only difference being the `.test.`

so in this case `index.test.js`